dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/utils/ReferenceConfigCache.java

```diff
@@ -115,12 +130,50 @@ public static ReferenceConfigCache getCache(String name, KeyGenerator keyGenerat
         return (T) config.get();
     }
 
+    public <T> void  put(String version,String url,String name,ReferenceConfig<T> referenceConfig){
+        if(StringUtils.isBlank(version)){
```

Comment:
You need to format your code, check https://github.com/apache/incubator-dubbo/blob/master/codestyle/dubbo_codestyle_for_idea.xml